### ISSUE TITLE
js_math_path tries to be smart, is actually dumb

### ISSUE THREAD

**Issue by shimizukawa** [sphinx-doc/sphinx#552]
```
jsmath_path = "http://wiki.tiker.net/jsmath/easy/load.js"
```

results in

```
... src="_static/http://wiki.tiker.net/jsmath/easy/load.js"
```

Ouch.

---
- Bitbucket: https://bitbucket.org/birkenfeld/sphinx/issue/552
- Originally reported by: Andreas Kloeckner
- Originally created at: 2010-11-02T04:02:57.119

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2010-11-12 07:25:26+00:00_

Not sure you filled the version field correctly; this was fixed in version 1.0.4.

**Comment by shimizukawa** [comment]
_From Georg Brandl on 2014-10-18 07:30:26+00:00_

Removing version: 1.0.4 (automated comment)
